### DISCUSSION TITLE
Changing or tweaking chess rules.

### DISCUSSION THREAD

**Discussion by HectorShaw** [official-stockfish/Stockfish#4633] [General]
So i wanted to integrate the engine for my portfolio project which i have made as my full stack app with web socket. But, have tweaked the chess rules by a bit will stock fish engine support the changes and if i plan to integrate what changes do i need to implement

**Comment by cj5716** [comment]
Have you looked at the fairy-stockfish project? Does it cover the variant you are supporting?

**Comment by HectorShaw** [comment]
Yes just checked it out even got the node module to work with. Thanks!
